packages/code-morph/src/morphers/tests/definition-schema-wrapper/complex/input.ts

```
@@ -0,0 +1,23 @@
+// @ts-nocheck
```

ESLint

[error] Do not use "@ts-nocheck" because it alters compilation errors.

(@typescript-eslint/ban-ts-comment)